packages/openapi-code-generator/package.json

```diff
@@ -24,8 +24,18 @@
     "test": "jest"
   },
   "devDependencies": {
+    "@azure-tools/typespec-autorest": "^0.41.1",
```

Comment:
These are just required by the integration tests, hence not included in `peerDependencies`